models/compositions/rock_control.rb

```diff
@@ -0,0 +1,42 @@
+require 'rock/models/services/motion2d_control_loop'
+require 'rock/models/services/pose'
+require 'models/services/distance_bearing_sensor'
+
+using_task_library 'controldev'
+using_task_library 'tut_follower'
+using_task_library 'tut_sensor'
+using_task_library 'rock_tutorial'
+
+module Tutorials
+    class RockControl < Syskit::Composition
+
+        add Controldev::JoystickTask, as: 'joystick'
```

Comment:
The models generated from the orogen components are now in the OroGen:: namespace. You can set `Roby.app.backward_compatible_naming = false` in the bundle's config/init.rb to disable the backward-compatible naming.